### PR TITLE
Feat noticias

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

**Comment by mixcas** [comment]
ready

File: src/native/routes/index.js [hunk 1]
```diff
@@ -27,86 +30,118 @@ import ProfileComponent from '../components/Profile';
...
+            <Scene key='noticias' component={NoticiasContainer} Layout={NoticiasComponent}/>
+          </Stack>
+        </Tabs>
+      </Scene>
+    </Stack>
+  </Scene>
+);
+
+/*
+<Stack
```
Comment (varffs) [inline src/native/routes/index.js]: I think all this can go

File: src/native/index.js [hunk 2]
```diff
@@ -15,26 +15,35 @@ import Loading from './components/Loading';
...
-  </Provider>
-);
+class Root extends Component {
+  render() {
+    const { store, persistor } = this.props;
 
+    return(
+      <Provider store={store}>
+      {/*
+        <PersistGate
```
Comment (varffs) [inline src/native/index.js:25]: wassup with this stuff?

File: src/containers/NoticiasContainer.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    match: null,
+  }
+
+  componentDidMount = () => this.fetchNoticias();
+
+  /**
+    * Fetch Data from API, saving to Redux
+    */
+  fetchNoticias = () => {
+    console.log('CONTAINER', this.props);
```
Comment (varffs) [inline src/containers/NoticiasContainer.js]: leftover logging

File: src/containers/NoticiasContainer.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { getNoticias, setError } from '../actions/noticiasActions';
+
+class NoticiaListing extends Component {
```
Comment (varffs) [inline src/containers/NoticiasContainer.js]: I kinda read this as something singular like the listing for each noticia not the list view for all. I'd think something like `NoticiasList` or `NoticiasFeed` would be better

File: src/containers/NoticiasContainer.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+  }
+
+  render = () => {
+    const { Layout, noticias, match } = this.props;
+    const id = (match && match.params && match.params.id) ? match.params.id : null;
+
+    return (
+      <Layout
+        noticiaId={id}
```
Comment (varffs) [inline src/containers/NoticiasContainer.js]: which noticia is this refering to?

File: src/native/components/Login.js [hunk 6]
```diff
@@ -42,10 +42,29 @@ class Login extends React.Component {
...
+        Actions.push('main');
+      })
       .catch(e => console.log(`Error: ${e}`));
   }
 
+  checkIfLogged(member) {
+    // Redirect if already logged in
+    if (member !== undefined && member.uid) {
+      Actions.push('main', this.props);
+    }
```
Comment (varffs) [inline src/native/components/Login.js]: would this function also redirect back out if not logged in?

File: src/native/components/Noticias.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  if (loading) return <Loading />;
+
+  // Error
+  if (error) return <Error content={error} />;
+
+  const keyExtractor = item => item.id;
+
+  const onPress = item => Actions.noticia({ match: { params: { id: String(item.id) } } });
+
+  const styles = StyleSheet.create({
```
Comment (varffs) [inline src/native/components/Noticias.js:30]: do you know if we can move away from this styling approach?

File: src/native/components/Noticias.js [hunk 8]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, FlatList, TouchableOpacity, RefreshControl, Image, View } from 'react-native';
+import { Container, Content, Card, CardItem, Body, Text, Button } from 'native-base';
+import { Actions } from 'react-native-router-flux';
+import getRNDraftJSBlocks from 'react-native-draftjs-render';
+import { distanceInWordsToNow } from 'date-fns';
```
Comment (varffs) [inline src/native/components/Noticias.js:7]: any particular reason to use this over moment?

File: src/native/components/Noticias.js [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+  });
+
+  return (
+    <Container>
+      <Content padder>
+        <Header
+          title="Noticias"
+        />
+
+        <FlatList
```
Comment (varffs) [inline src/native/components/Noticias.js:45]: should we be using these prebuilt components? fine for now but a question

**Review by varffs — CHANGES_REQUESTED** [review]
I think we could do with lots more commenting esp on unfamiliar stuff. I'm pretty sure I get it and happy to merge once the minor bigs like excess logs and comments are deleted. However a few questions to answer too.

File: src/native/components/Noticias.js (hunk 8 above)
Comment (mixcas) [reply src/native/components/Noticias.js:7]: They are different, `date-fns` is like underscore for dates. While moment is kinda like jQuery for dates.

File: src/native/components/Noticias.js (hunk 7 above)
Comment (mixcas) [reply src/native/components/Noticias.js:30]: idk, this is the native way

File: src/native/components/Noticias.js (hunk 9 above)
Comment (mixcas) [reply src/native/components/Noticias.js:45]: yes, is the native way. we could use a library that uses other predefined components but they would be built on top of this anyay.

File: src/native/index.js (hunk 2 above)
Comment (mixcas) [reply src/native/index.js:25]: > Redux Persist takes your redux state object and saves it to persisted storage. On app launch, it retrieves this persisted state and saves it back to redux.

Disabled for developing for now.

File: src/containers/NoticiasContainer.js (hunk 5 above)
Comment (mixcas) [reply src/containers/NoticiasContainer.js]: `id` comes from line 43

Comment (mixcas) [reply src/containers/NoticiasContainer.js]: ummmm actually i think that shouldn't be there

File: src/native/components/Login.js (hunk 6 above)
Comment (mixcas) [reply src/native/components/Login.js]: i dont understand ur question, explan

**Comment by varffs** [comment]
@mixcas is this still PR live? If so plz answer the comment questions

**Review by mixcas — COMMENTED** [review]
Seems like i left a "review" with comments and everything pending. My bad

File: src/native/components/Noticias.js (hunk 8 above)
Comment (varffs) [reply src/native/components/Noticias.js:7]: The repo for `date-fns` clearly describes it as a replacement for moment?

Comment (mixcas) [reply src/native/components/Noticias.js:7]: i like this better cuz has better syntax and uses pure functions unlike moment were u have to create objects and stuff

**Pull request merged**